cert/util.go

```diff
@@ -12,27 +14,6 @@ import (
 	"strings"
 )
 
-// CompareCertificates x509 compares two CA certificates
-func CompareCertificates(cert1, cert2 []byte) (bool, error) {
-	p1, _ := pem.Decode(cert1)
-	if p1 == nil {
-		return false, errors.New("Unable to pem decode certificate")
-	}
-	parsedCert1, err := x509.ParseCertificate(p1.Bytes)
-	if err != nil {
-		return false, err
-	}
-	p2, _ := pem.Decode(cert2)
-	if p2 == nil {
-		return false, errors.New("Unable to pem decode certificate")
-	}
-	parsedCert2, err := x509.ParseCertificate(p2.Bytes)
-	if err != nil {
-		return false, err
-	}
-	return parsedCert1.Equal(parsedCert2), nil
-}
-
 func displayName(name pkix.Name) string {
```

Comment:
Why aren't we just using the pkix.Name String function here? https://golang.org/pkg/crypto/x509/pkix/#Name